modules/js/src/embindgen.py

```diff
@@ -155,9 +103,16 @@ def makeWhiteList(module_list):
                 wl[k] = m[k]
     return wl
 
-white_list = makeWhiteList([core, imgproc, objdetect, video, dnn, features2d, calib3d])
+track = ""
+if "OPENCV_JS_WHITELIST" in os.environ:
+    track = os.environ["OPENCV_JS_WHITELIST"]
+else:
+    track = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'opencv_js.config.py')
```

Comment:
Let's take the path from `OPENCV_JS_WHITELIST` anyway. Please replace default value `opencv_js.config.py` to build_js.py